pkg/node/volumemgr.go

```diff
@@ -814,6 +824,13 @@ func (m *VolumeManager) discoverDataOnDrives() error {
 	return nil
 }
 
+func (m *VolumeManager) isLogChange(logMessage, driveID string) bool {
```

Comment:
I don't think this is a good idea to store log and compare it

Reply:
I rolled back the changes and converted this message to debug. Also this message will be displayed when the `kubectl logs` is called.